code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -6,8 +6,8 @@
 	flags_cover = MASKCOVERSMOUTH
 	w_class = WEIGHT_CLASS_SMALL
 	gas_transfer_coefficient = 0.90
-	put_on_delay = 20
-	var/resist_time = 0 //deciseconds of how long you need to gnaw to get rid of the gag, 0 to make it impossible to remove
+	put_on_delay = 2 SECONDS 
+	var/resist_time = 0 SECONDS // How long you need to gnaw to get rid of the gag, 0 to make it impossible to remove
```

Comment:
```suggestion
	/// How long you need to gnaw to get rid of the gag, 0 to make it impossible to remove
	var/resist_time = 0 SECONDS
```
I specifically requested it to be here so autodoc works

Reply:
I can't say how this fixes or bricks an autodoc, but if you say so.

Reply:
Autodoc for vars is done with `/// Comment` on the line above the var